docs/using/mongodb_limitations.md

```diff
@@ -0,0 +1,94 @@
+# MongoDB Backup and Restore support matrix
+
+Creating and restoring MongoDB backups in PMM currently has the following limitations and requirements:
+
+- Restoring on different Replica set/Cluster is not supported.
+- Physical restores are not supported for deployments with arbiter nodes. For more information, see the [Percona Backup for MongoDB documentation](https://docs.percona.com/percona-backup-mongodb/usage/restore.html#physical-restore-known-limitations).
+- All types of backups on sharded cluster setups are currently not supported.
+- Physical backups are supported, but restoring from physical backups will cause all mongo and pbm-agent instances to shutdown. To bring them back up, restart all **mongod** and **mongos** nodes and restart all **pbm-agents**. 
+- Retention policy is supported only for snapshot types of scheduled backups.
+- Before restoring, make sure to prevent clients from accessing database.
+
+
+
+
+## Support matrix
+
+### Logical Backup/Restore
+
+|    Storage type           | Full   | PITR    |
+| ------------- | ------ | ------- |
+| Local storage | **Backup**/**Restore** |**Backup**/ - |
+| S3            | **Backup**/**Restore** | **Backup**/**Restore** |
+
+### Physical Backup/Restore
+
+|      Storage type          | Full   | PITR      |
+| ------------- | ------ | --------- |
+| Local storage | **Backup**/**Restore.** Restoring requires manual restart of mongod and pbm-agents. | -/-|
+| S3            | **Backup**/**Restore**. Restoring requires manual restart of mongod and pbm-agents. |  -/-|
+
+
+### Full matrix
+
+<style>
+  table th:first-of-type {
+    width: 5%
+  }
+  table th:nth-of-type(2) {
+    width: 5%
+  }
+  table th:nth-of-type(3) {
+    width: 2%
+  }
+  table th:nth-of-type(4) {
+    width: 2%
+  }
+  table th:nth-of-type(5) {
+    width: 2%
+  }
+  table th:nth-of-type(6) {
+    width: 2%
+      }
+  table th:nth-of-type(7) {
+    width: 10%
+  }
+  table td {
+    text-align:left; vertical-align: top;
+  }
+</style>
+
+| Backup/Restore | Logical/Physical | PITR | S3/Local | Containerized | Support level| Comments                                                                      |
```

Comment:
I think we should remove `Comments` column from this doc or at least remove some remarks to not confuse users.